src/test/resources/io/jenkins/plugins/gitlabserver/casc/configuration-as-code.yml

```diff
@@ -0,0 +1,16 @@
+credentials:
+  system:
+    domainCredentials:
+      - credentials:
+          - gitlabPersonalAccessToken:
+              scope: SYSTEM
+              id: "i<3GitLab"
+              token: "XfsqZvVtAx5YCph5bq3r"
```

Comment:
Can we use some method to populate the token via enviroment variable?

Reply:
Yes
https://github.com/jenkinsci/configuration-as-code-plugin#handling-secrets

Reply:
Yes, I read it. I want to know if we can populate the env variable via API or some method without going to the UI.